src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,98 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private final RegistrationService validateRegistration = new RegistrationServiceImpl();
+    private final User validUser = new User();
+    private final User validUser2 = new User();
+    private final StorageDao data = new StorageDaoImpl();
+
+    @Test
+    void age_isOk() {
```

Comment:
in all of the namings don't forget to add the name of the tested method at the beginning